Content.Server/Chat/Systems/ChatSystem.cs

```diff
@@ -717,6 +717,8 @@ private string SanitizeInGameICMessage(EntityUid source, string message, out str
 
         _sanitizer.TrySanitizeOutSmilies(newMessage, source, out newMessage, out emoteStr);
 
+        newMessage = FormattedMessage.EscapeText(newMessage);
```

Comment:
We already have code in the rest of the chat system to sanitize out rich text tags, you are now sanitizing it twice, in a different way. Because the other system does it by stripping rather than escaping I would encourage you to check WHY this is the case, and if there is no risk of a system starting to output rich text tags to IC you should remove the other sanitizations as well.

Reply:
Just revert the change, this system is a mess and it's better to leave it for chat refactor.

Reply:
Ok